ui-admin/src/study/StudyEnvironmentRouter.tsx

```diff
@@ -174,6 +177,11 @@ export const studyEnvNotificationsPath = (portalShortcode: string, studyShortcod
   return `/${portalShortcode}/studies/${studyShortcode}/env/${envName}/notificationContent`
 }
 
+/** helper for path to configure study messages */
+export const studyEnvMessagesPath = (portalShortcode: string, studyShortcode: string, envName: string) => {
```

Comment:
Are messages the same as alerts?

Reply:
In the first few commits in this PR I originally called these "messages" rather than "alerts". They're the same, but looks like this usage slipped through the cracks when renaming.

I'm indifferent about what to call these, but if there are strong feelings about going back to "messages" I'm happy to make that change now while the concept is still quarantined to this PR

Reply:
Thx. I do not think changing everything from 'alert' to 'message' it is worth the trouble since the value is debatable.
It would be nice to use a consistent term. Perhaps changing these to 'alert' is something for a future PR?

Reply:
Since the code was already in front of me, I changed these to "alert"